### PR TITLE
Added user context for createdby

### PR DESCRIPTION
**Pull request by mindhash**

### CODE REVIEW

File: pkg/query-service/auth/jwt.go [hunk 1]
```diff
@@ -92,3 +92,15 @@ func ExtractJwtFromContext(ctx context.Context) (string, error) {
...
+// ExtractUserFromContext extracts user from context.Context using accssJwt. here,
+// we do not validate if user exists as it would be done so while adding
+// user to the context
+func ExtractUserFromContext(ctx context.Context) (*model.UserPayload, error) {
+	jwt, err := ExtractJwtFromContext(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	return validateUser(jwt)
```
Comment (nityanandagohain) [inline pkg/query-service/auth/jwt.go:105]: Not required as the jwt is already validated in the authentication process.

**Review by nityanandagohain — COMMENTED** [review]
This look good but can you please add

```
created_at,
COALESCE((SELECT NAME FROM users 
 WHERE id = v.created_by), "unknown") created_by_name, 
```

in GetConfigHistory, GetConfigVersion, GetLatestVersion

File: pkg/query-service/auth/jwt.go (hunk 1 above)
Comment (mindhash) [reply pkg/query-service/auth/jwt.go:105]: agree. But no harm in re-using the common method.  the method does not have any db calls

Comment (nityanandagohain) [reply pkg/query-service/auth/jwt.go:105]: Cool, can you make the changes the Get functions as requested ?

**Pull request merged**